src/schema/ghga.yaml

```diff
@@ -604,20 +605,18 @@ classes:
           - EFO:0000635
       has disease:
         description: >-
-          The Disease entity that is associated with this Biospecimen at the time of retrieval
-          from the organism. Typically, a concept from Mondo Disease Ontology.
-          For example, 'MONDO:0003742' indicates that the Individual - from which the Biospecimen
-          was extracted from - suffers from 'Heart Fibrosarcoma'.
+          The Disease entity that is associated with the Individual. Typically, a concept
+          from Mondo Disease Ontology. For example, 'MONDO:0005267' indicates that
```

Comment:
Should we suggest to also use EFO for the phenotypic feature?
That is what EGA suggests, but they are also accepting other ontologies.

Reply:
Hmmm, yeah. I don't think we are constraining ourselves to just Mondo. This is just an example :)